[2025-08-15T17:16:11.145+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-15T17:16:11.187+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: orders_pipeline.upload_to_s3 manual__2025-08-15T17:16:07.715886+00:00 [queued]>
[2025-08-15T17:16:11.197+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: orders_pipeline.upload_to_s3 manual__2025-08-15T17:16:07.715886+00:00 [queued]>
[2025-08-15T17:16:11.198+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-15T17:16:11.219+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): upload_to_s3> on 2025-08-15 17:16:07.715886+00:00
[2025-08-15T17:16:11.228+0000] {standard_task_runner.py:64} INFO - Started process 56 to run task
[2025-08-15T17:16:11.236+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'orders_pipeline', 'upload_to_s3', 'manual__2025-08-15T17:16:07.715886+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/orders_pipeline.py', '--cfg-path', '/tmp/tmpg6f4qm3t']
[2025-08-15T17:16:11.239+0000] {standard_task_runner.py:91} INFO - Job 46: Subtask upload_to_s3
[2025-08-15T17:16:11.324+0000] {task_command.py:426} INFO - Running <TaskInstance: orders_pipeline.upload_to_s3 manual__2025-08-15T17:16:07.715886+00:00 [running]> on host 70dc4360fa72
[2025-08-15T17:16:11.525+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='orders_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-08-15T17:16:07.715886+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-15T17:16:07.715886+00:00'
[2025-08-15T17:16:11.527+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-15T17:16:11.551+0000] {credentials.py:1217} INFO - Found credentials in environment variables.
[2025-08-15T17:16:12.824+0000] {python.py:237} INFO - Done. Returned value was: s3://pt-data-lab/raw-data/orders/dt=2025-08-15/orders.csv
[2025-08-15T17:16:12.825+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-15T17:16:12.863+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=orders_pipeline, task_id=upload_to_s3, run_id=manual__2025-08-15T17:16:07.715886+00:00, execution_date=20250815T171607, start_date=20250815T171611, end_date=20250815T171612
[2025-08-15T17:16:12.908+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-08-15T17:16:12.939+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-15T17:16:12.942+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
